src/gatsby-node.ts

```diff
@@ -149,7 +149,7 @@ export const createSchemaCustomization: GatsbyNode['createSchemaCustomization']
         rawContent: 'String!',
         shortDescription: 'String!',
         thumbnail: {
-          type: 'String',
+          type: 'File',
```

Comment:
This is actually not the proper type for the source plugin. because the only thing I can get from the API surface is a string or URL.

You can derive it on your site node, or transformer as you need.

Reply:
However, thumbnails of `VelogUser`, `VelogTag`, and `VelogSeries` types are File type. Not only that, as you can see that `createRemoteFileNode.id` is assigned to `post.thumbnail`, this seems close to the original intention.

```
if (post.thumbnail) {
  const thumbnailNode = await createRemoteFileNode({
    url: post.thumbnail,
    store,
    cache,
    reporter,
    createNode,
    createNodeId,
  });
  postSource.thumbnail = thumbnailNode.id;
} else {
  delete postSource['thumbnail'];
}
```

Reply:
Additionally, `thumbnail` of `String` is returning `null`.

<img width="1604" alt="image" src="https://user-images.githubusercontent.com/5603687/200215661-50810c15-a4ec-4ddf-9601-987e0e841819.png">

Reply:
Ah you right. I totally forgot I intented ease of use on this plugin. Maybe I should refactor those types 🤔